src/giskardpy/plugin_world_visualization.py

```diff
@@ -67,19 +66,23 @@ def update(self):
         for object_name, object in objects_dict.items():
             for link_name in object.get_link_names():
                 if object.has_link_visuals(link_name):
-                    marker = object.link_as_marker(link_name)
-                    if marker is None:
+                    # Simple objects (containing only one link) are published here:
+                    if link_name == object_name and len(object.get_link_names()) == 1:
+                        marker = object.as_marker_msg()
+                        markers.append(marker)
```

Comment:
Doesn't that mean that the ```marker.id``` and so on are not being set anymore? Is that intentional?